EDDiscovery/Forms/Form2DMap.cs

```diff
@@ -88,37 +75,44 @@ private void FormSagCarinaMission_Load(object sender, EventArgs e)
                 return;
             }
 
-            toolStripComboBox1.Items.Clear();
+            toolStripComboExpo.Items.Clear();
 
             foreach (Map2d img in fgeimages)
             {
-                toolStripComboBox1.Items.Add(img.FileName);
+                toolStripComboExpo.Items.Add(img.FileName);
             }
-            
-            toolStripComboBox1.SelectedIndex = 0;
+
+            toolStripComboBoxTime.Items.AddRange(new string[] {
+            "Distant Worlds Expedition",
+            "FGE Expedition start",
+            "Last Week",
+            "Last Month",
+            "Last Year",
+            "All",
+            "Custom"});
+
+            toolStripComboExpo.SelectedIndex = 0;
             toolStripComboBoxTime.SelectedIndex = 0;
             initdone = true;
             ShowSelectedImage();
+
+            imageViewer.BackColor = Color.FromArgb(5, 5, 5);
+
+            EDDiscovery.EDDTheme theme = EDDiscovery.EDDTheme.Instance;
+            bool winborder = theme.ApplyToForm(this);
+            statusStripCustom.Visible = panel_close.Visible = panel_minimize.Visible = !winborder;
```

Comment:
Probably a good idea to add a base class `protected ObservableList<Control> CaptionControls {get;}` or a `protected void Add/Remove-To/From-Caption(Control c) {}` to handle this stuff upstream.